Packs/CohesityHelios/Integrations/CohesityHelios/CohesityHelios.py

```diff
@@ -0,0 +1,485 @@
+
+    return {
+        "alert_id": alert['id'],
+        "occurrence_time": occurance_time,
+        "severity": alert.get('severity'),
+        "alert_description": alert['alertDocument']['alertDescription'],
+        "alert_cause": alert['alertDocument']['alertCause'],
+        "anomalous_object_name": property_dict.get('object'),
+        "anomalous_object_env": property_dict.get('environment'),
+        "anomaly_strength": property_dict.get('anomalyStrength')
+    }
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def get_ransomware_alerts_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Gets ransomware alerts detected by Cohesity Helios.
+
+        :type client: ``Client``
+        :param Client:  cohesity helios client to use.
+
+        :type args: ``Dict[str, Any]``
+        :param args: Dictionary with get ransomware alerts parameters.
+
+    Returns command result with the list of fetched ransomware alerts.
+    """
+    start_time_millis = datestring_to_millis(args.get('created_after', ''))
+    end_time_millis = datestring_to_millis(args.get('created_before', ''))
+    severity_list = args.get('alert_severity_list', None)
+    ids_list = args.get('alert_id_list', None)
+    limit = args.get('limit', MAX_FETCH_DEFAULT)
+
+    # Fetch ransomware alerts via client.
+    resp = client.get_ransomware_alerts(
+        start_time_millis=start_time_millis,
+        end_time_millis=end_time_millis, alert_ids=ids_list,
+        alert_severity_list=severity_list,
+        max_fetch=limit)
+    demisto.debug("Got {numAlerts} alerts between {start} and {end}".
+                  format(numAlerts=len(resp), start=start_time_millis, end=end_time_millis))
+
+    # Parse alerts for readable output.
+    ransomware_alerts = []
+    for alert in resp:
+        alert_details = get_ransomware_alert_details(alert)
+        ransomware_alerts.append(alert_details)
+
+    return CommandResults(
+        outputs_prefix='CohesityHelios.RansomwareAlert',
+        outputs_key_field='alert_id',
+        outputs=ransomware_alerts,
+    )
+
+
+def ignore_ransomware_anomaly_command(client: Client, args: Dict[str, Any]) -> str:
+    """Ignore detected anomalous object on Helios.
+
+        :type client: ``Client``
+        :param Client:  cohesity helios client to use.
+
+        :type args: ``Dict[str, Any]``
+        :param args: Dictionary with ignore anomaly parameters.
+
+        :return: success message of the ignore anomaly operation.
+        :rtype: ``str``
+    """
+    # Filter ransomware alert for given object name.
+    alert_id = ''
+    object_name = args.get('object_name')
+    demisto.debug("Performing ignore anomaly operation for object {name}".format(name=object_name))
+
+    resp = client.get_ransomware_alerts()
+    for alert in resp:
+        property_dict = _get_property_dict(alert['propertyList'])
+        if property_dict.get('object', "") == object_name:
+            alert_id = alert.get('id')
```

Comment:
Shouldn't you `break` the for loop after finding alert_id"?

Reply:
done